eth2util/hash.go

```diff
@@ -40,3 +41,14 @@ func SlotHashRoot(slot eth2p0.Slot) ([32]byte, error) {
 
 	return hash, nil
 }
+
+// SHA256 calculates the sha256 checksum of the input data. https://github.com/ethereum/consensus-specs/blob/v0.9.3/specs/core/0_beacon-chain.md#hash
+func SHA256(data []byte) [32]byte {
+	h := sha256.New()
+	h.Write(data)
+
+	var b [32]byte
+	h.Sum(b[:0])
+
+	return b
```

Comment:
```suggestion
	return h.Sum(nil)
```

Reply:
we declare the 32byte array as the return type because the output of SHA256 is an 32 bytes array